### PR TITLE
Circos fixes

### PR DESCRIPTION
**Pull request by hexylena**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

- replaces `&&` with `;`  (in conjunction with tool profile)
- fixes export of solely configuration archive
- updates dependencies
- adds some new test cases based on user reports (thanks @erxleben)

### CODE REVIEW

**Comment by hexylena** [comment]
@gallardoalba maybe you'd be interested in reviewing some additional updates I made to circos (mostly updating deps, adding a tool profile..)

File: tools/circos/circos.xml [hunk 1]
```diff
@@ -18,6 +13,11 @@
     <edam_operations>
         <edam_operation>operation_0337</edam_operation>
     </edam_operations>
+    <xrefs>
+        <xref type="bio.tools">galactic_circos</xref>
+        <!--<xref type="bioconda">circos</xref>-->
```
Comment (bgruening) [inline tools/circos/circos.xml:18]: Do they fail during linting?

**Review by bgruening — APPROVED** [review]

File: tools/circos/circos.xml (hunk 1 above)
Comment (hexylena) [reply tools/circos/circos.xml:18]: I don't know why that one is commented out.

Comment (hexylena) [reply tools/circos/circos.xml:18]: Yep, they do.

```
.. ERROR: Invalid XML found in file: circos.xml. Errors [/tmp/tmprtvkjkzc:12:0:ERROR:SCHEMASV:SCHEMAV_CVC_ENUMERATION_VALID: Element 'xref', attribute 'type': [facet 'enumeration'] The value 'bioconda' is not an element of the set {'bio.tools', 'bioconductor'}.
/tmp/tmprtvkjkzc:13:0:ERROR:SCHEMASV:SCHEMAV_CVC_ENUMERATION_VALID: Element 'xref', attribute 'type': [facet 'enumeration'] The value 'rrid' is not an element of the set {'bio.tools', 'bioconductor'}.]
```

**Comment by hexylena** [comment]
> System.IO.IOException: No space left on device : '/home/runner/runners/2.303.0/_diag/Worker_20230323-203631-utc.log'

oops

**Comment by bgruening** [comment]
I tried it now multiple times, there is something wrong.

**Comment by bernt-matthias** [comment]
Hey @hexylena can you create a new PR adding `3/chrom.tab` .. seems that the tests only appeared to be successful

**Comment by hexylena** [comment]
ja türlich

**Pull request merged**